eclair-core/src/main/scala/fr/acinq/eclair/db/migration/MigratePeersDb.scala

```diff
@@ -0,0 +1,41 @@
+package fr.acinq.eclair.db.migration
+
+import fr.acinq.eclair.db.jdbc.JdbcUtils.ExtendedResultSet._
+import fr.acinq.eclair.db.migration.MigrateDb.{checkVersions, migrateTable}
+
+import java.sql.{Connection, PreparedStatement, ResultSet}
+
+object MigratePeersDb {
+
+  private def migratePeersTable(source: Connection, destination: Connection): Int = {
+    val sourceTable = "peers"
+    val insertSql = "INSERT INTO local.peers (node_id, data) VALUES (?, ?)"
+
+    def migrate(rs: ResultSet, insertStatement: PreparedStatement): Unit = {
+      insertStatement.setString(1, rs.getByteVector("node_id").toHex)
+      insertStatement.setBytes(2, rs.getBytes("data"))
+    }
+
+    migrateTable(source, destination, sourceTable, insertSql, migrate)
+  }
+
+  private def migrateRelayFeesTable(source: Connection, destination: Connection): Int = {
+    val sourceTable = "relay_fees"
+    val insertSql = "INSERT INTO local.relay_fees (node_id, fee_base_msat, fee_proportional_millionths) VALUES (?, ?, ?)"
+
+    def migrate(rs: ResultSet, insertStatement: PreparedStatement): Unit = {
+      insertStatement.setString(1, rs.getByteVector("node_id").toHex)
+      insertStatement.setLong(2, rs.getLong("fee_base_msat"))
+      insertStatement.setLong(3, rs.getLong("fee_proportional_millionths"))
+    }
+
+    migrateTable(source, destination, sourceTable, insertSql, migrate)
+  }
+
+  def migrateAllTables(source: Connection, destination: Connection): Unit = {
+    checkVersions(source, destination, "peers", 2, 3)
```

Comment:
By hard-coding the values here instead of using `SqlitePeersDb.CURRENT_VERSION` and `PgPeersDb.CURRENT_VERSION`, you ensure that this migration will fail in future versions of eclair where the DB versions change.

That's expected, right? Because we would need to re-write the migration as they are version-specific?
Users can only do the Sqlite -> Postgres migration once, when we release this eclair version.
Then if we want to support migrating in next eclair versions, we will update the code here.

Reply:
Exactly. It's pretty convenient and allowed me to quickly upate the migration code for the audit and peers db when preparing this PR.